content/starter-configs/tutorials/wordpress.md

```diff
@@ -48,62 +50,106 @@ define('DB_PASSWORD', 'tugboat');
+to tweak a few things for your particular installation.
+
+For each service (php, mysql, etc) Tugboat runs three phases to build your preview:
+[init, update, and build](/building-a-preview/preview-deep-dive/how-previews-work/#the-build-process-explained).
+
+### Phase 1: INIT
+
+In the INIT phase, Tugboat sets up your server and creates the docker containers for your services.
+
+1. If you're using composer to install Wordpress Core, uncomment that line in the `init` phase.
+2. Just after that, make sure that you're mapping the `${DOCROOT}` to whatever you named the directory where you
+   downloaded Wordpress.
+
+### Phase 2: UPDATE
+
+In the UPDATE phase, the server is ready, and you can now import any other assets your site needs, like images, a
+database, etc.
+
+1. If you're importing file assets, change the paths to match where you're importing them from.
+
+### Phase 3: BUILD
+
+In the BUILD phase, run any commands that your site needs to prepare itself to run.
+
+1. Nothing to customize from this starter template.
+
+### The `mysql` Service
+
+At the bottom, in the `mysql` service, update the settings to fetch a database from a remote source and import it into
+Tugboat.
+
+### Wordpress Starter Config
+
+Copy this into `.tugboat/config.yml`
 
 ```yaml
 services:
   # What to call the service hosting the site.
   php:
-    # Use PHP 7.x with Apache to serve the WordPress site; this syntax pulls in the latest version of PHP 7
-    image: tugboatqa/php:7-apache
+    # Use PHP 8.x with Apache to serve the WordPress site; this syntax pulls in the latest version of PHP 8.1
+    image: tugboatqa/php:8.1-apache
 
     # Set this as the default service. This does a few things
     #   1. Clones the git repository into the service container
     #   2. Exposes port 80 to the Tugboat HTTP proxy
     #   3. Routes requests to the preview URL to this service
     default: true
 
-    # Wait until the mysql service is done building
+    # Wait until the mysql service is done building so we have a database.
     depends: mysql
 
-    # A set of commands to run while building this service
+    # A set of commands to run while building this service.
     commands:
-      # Commands that set up the basic preview infrastructure
+      # Phase 1 (init): Configure the server infrastructure.
       init:
-        # Install prerequisite packages
+        # Install prerequisite packages.
         - apt-get update
-        - apt-get install -y rsync libzip-dev
+        - apt-get install -y rsync
 
-        # Enable extra apache modules.
-        - a2enmod rewrite headers
+        # Turn on URL rewriting.
+        - a2enmod rewrite
 
-        # Install the PHP extensions
-        - docker-php-ext-install mysqli exif zip
-        # Install imagick
-        - apt-get install -y libmagickwand-dev
-        - pecl install imagick-beta -y
-        - docker-php-ext-enable imagick
```

Comment:
Does WordPress not use Imagemagick?

Reply:
I think these came out from an old demo file I accidentally started with.  They're back in now.